packages/backend/src/studio-api-impl.spec.ts

```diff
@@ -18,63 +18,122 @@
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import { expect, test, vi } from 'vitest';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
 import content from './ai-test.json';
+import userContent from './ai-user-test.json';
```

Comment:
couldn't you just use the `ai-test.json` file? So we avoid having multiple files for just testing.
In my case i just check the name, registry, url. So if you prefer your content it's just a matter of updating few lines in the old tests and removing the other json.

Reply:
What I want to test is that the user's catalog is used, when it is present. For this, I need a second, different, catalog, to be sure by reading it that it has been loaded.

Reply:
My bad, by opening it on vscode and seeing the code without any annotation it was clear 👍